tests/integration_tests/test_image_data_tasks.py

```diff
@@ -45,5 +46,16 @@ def test_image_classification():
     assert(history.history['val_acc'][-1] > 0.85)
 
 
+@keras_test
+def test_directory_iterator():
+    generator = ImageDataGenerator()
+    directory_iterator = generator.flow_from_directory('./tests/integration_tests/images')
```

Comment:
Do not include images in the git tree. If you need images, generate them. We have tools for that in `preprocessing.image`.